modules/microshift-storage-device-classes.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/creating-volume-snapshots-microshift.adoc
+
+:_content-type: CONCEPT
+[id="microshift-storage-device-classes_{context}"]
+== Device classes
```

Comment:
I thought we were not going to include discussion of multiple device classes in the thin provisioning instructions. Maybe I'm misremembering? This content isn't wrong, but maybe we can move it to a chapter for managing multiple device classes instead of having it in the snapshot chapter.

Reply:
No, there was no mention of not including this. Removing some devices was a goal. We did remove raid.

Reply:
Could be my ignorance of adoc, but isn't this file already it's own section for device-classes?

IIRC, the consensus was to provide a simple example lvmd.yaml for snapshotting, but also provide an expanded config like the one here.

Reply:
Yes, this is an individual module. It can be called in any assembly. 

We have another open docs ticket for "day 2" storage config operations. This could go there once that is established--or in an existing section if it makes more sense. OCP weaves this content into each provider, which isn't relevant here. I don't where else it goes in the workflow for a MicroShift admin?

Reply:
https://docs.google.com/document/d/1Slfy4-oHf7RqvP2tVnr6z6jHXwMHJssdOpw-G4zwUts/edit?disco=AAAA1e6X9sw is where we talked about this on the earlier draft.

Including this module in the thin provisioning instructions is mixing unrelated content. We should move it to its own chapter of the storage book.

Reply:
It's in the LVMS plugin section in OCP, so I parked it there for the moment. I think we can tie that module in with the other work around advanced config to come in a later PR. Sound good?